src/vuex/modules/extension.js

```diff
@@ -0,0 +1,28 @@
+import Vue from "vue"
+import config from "src/config"
+
+export default () => {
+  const emptyState = {
+    enabled: false
+  }
+  const state = {
+    ...emptyState,
+    externals: { config } // for testing
+  }
+  const mutations = {
+    setExtensionAvailable(state, enabled) {
+      Vue.set(state, "enabled", enabled)
+    }
+  }
+
+  const actions = {
+    async checkForPresense() {
+      // TODO Check if extension still installed. User could have uninstalled it.
```

Comment:
True, but that would be very rare case. Do we really need to check this?

Reply:
We should check… somewhere. Doesn't have to be here, or now.

Should we keep the module?

Reply:
I am open to cleaning up later.